sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/unresolved.scala

```diff
@@ -423,6 +423,26 @@ case class UnresolvedAlias(
 }
 
 /**
+ * Aliased column names for subquery. We could add alias names for output columns in the subquery:
+ * {{{
+ *   // Assign alias names for output columns
+ *   SELECT col1, col2 FROM testData AS t(col1, col2);
+ * }}}
+ *
+ * @param outputColumnNames the column names for this subquery.
+ * @param child the logical plan of this subquery.
```

Comment:
Nit: `the logical plan of this subquery` -> `the [[LogicalPlan]] on which this subquery column aliases apply`